services/QuillCMS/app/workers/bulk_reindex_responses_worker.rb

```diff
@@ -0,0 +1,20 @@
+class BulkReindexResponsesWorker
+  include Sidekiq::Worker
+  sidekiq_options retry: 5, queue: 'low'
+
+  def client
+    @client = Elasticsearch::Client.new url: ENV['ELASTICSEARCH_URL']
+  end
+
+  def perform(starting_index, batch_size)
+    batch_for_bulk = []
```

Comment:
This would be a good place to add logic for [cancelling jobs](https://github.com/mperham/sidekiq/wiki/FAQ#how-do-i-cancel-a-sidekiq-job)  It's much easier to add this logic and get it working now then later with the full dataset.